source/grammar/openpulse1.g4

```diff
@@ -0,0 +1,74 @@
+/**** ANTLRv4  grammar for OpenPulse1.0.
+
+The following grammar specifies valid OpenPulse statements within the .*? of
+OpenQASM3 below.
+
+calibrationDefinition
+    : 'defcal' calibrationGrammar? Identifier
+    ( LPAREN calibrationArgumentList? RPAREN )? identifierList
+    returnSignature? LBRACE .*? RBRACE  // for now, match anything inside body
+    ;
+
+specifically, `.*? -> body` below.
+
+****/
+
+grammar openpulse1;
+
+body
+    : (statement)+
+    ;
+
+statement
+    : classicalAssignment
+    | classicalDeclaration
+    | constantDeclaration
+    | aliasStatement
+    | branchingStatement
+    | instruction
+    | pragma
+    ;
+
+instruction
+    : 'play' LPAREN pulse RPAREN SEMICOLON
```

Comment:
You can also play a function or an identifier representing a function/array, right? Also need to include `frame` to play on.